content/ai/concepts/hyperparametertuning/terms/bayesianoptimization/bayesianoptimization.md

```diff
@@ -0,0 +1,33 @@
+---
+Title: 'Bayesian Optimizaztion'
+Description: 'Bayesian hyperparameter optimization encompasses algorithms used in the tuning phase of machine learning modelling that uses the scores from previous hyperparameter configurations to inform the selection of the next set.'
+Subjects:
+  - 'Machine Learning'
+  - 'Data Science'
+Tags:
+  - 'AI'
+CatalogContent:
+  - 'machine-learning'
+  - 'paths/data-science'
+---
+
+**Bayesian optimization** is a method employed in a the hyperparameter tuning process of machine learning. It uses previous evalutations of the objective function (the trained model score) to inform the selection of the next set of hyperparemeters to investigate.
+
+## Why Use Bayesian Optimization?
+
+With the effective simplicity of grid search and random search, it might seem unnecessary to explore alternatives when it comes to hyperparameter tuning. So what is the benefit to using Bayesian optimization over more basic methods? The answer is computational cost: by slightly increasing the computational time of selecting the next hyperparameter configuration, we can reduce the number of times that the model needs to be fit. This can be an invaluable method for instances where model complexity reaches a point in which fitting each model demands high computational costs and thus can take a long time.
```

Comment:
```suggestion
With the effective simplicity of grid search and random search, it might seem unnecessary to explore alternatives when it comes to hyperparameter tuning. So what is the benefit to using Bayesian optimization over more basic methods? The answer is computational cost: by slightly increasing the computational time of selecting the next hyperparameter configuration, it can reduce the number of times that the model needs to be fit. This can be an invaluable method for instances where model complexity reaches a point in which fitting each model demands high computational costs and thus can take a long time.
```